docs/source/ingest/destination_connectors/chroma.rst

```diff
@@ -0,0 +1,75 @@
+Chroma
+======================
+
+Batch process all your records using ``unstructured-ingest`` to store structured outputs locally on your filesystem and upload those to a Chroma database.
+
+First you'll need to install the Chroma dependencies as shown here.
+
+.. code:: shell
+
+  pip install "unstructured[chroma]"
+
+Run Locally
+-----------
+The upstream connector can be any of the ones supported, but for convenience here, showing a sample command using the
+upstream local connector. This will create new files on your local. And then upload those to Chroma.
+
+.. tabs::
```

Comment:
We seem to be having a new convention here where we render code from other folders:
https://github.com/Unstructured-IO/unstructured/blob/main/docs/source/ingest/destination_connectors/azure.rst

But it doesn't seem to be working / rendering for now so I'm not sure what to do here. I'd say this one is fine as well, just a note

Reply:
This will have to match the new doc format where we break out the .py and .sh Thanks for the reminder